src/Tools/ReleaseNotesUtil/Program.cs

```diff
@@ -171,6 +171,7 @@ private static IEnumerable<string> GetNetAnalyzerBinaries(string nugetInstalledP
             }
 
             string[] roslynAnalyzerPackages = new string[] {
+                "Microsoft.CodeAnalysis.NetAnalyzers",
```

Comment:
The rest of the packages are no longer maintained and can be deleted, I think?
Instead, consider adding `Microsoft.CodeAnalysis.PublicApiAnalyzers` and `Microsoft.CodeAnalysis.BannedApiAnalyzers`?

Can you also try to run the tool with both `getrulesjson` and `diffrules` and show a sample output? Just to make sure the tool is working correctly :)